guide/interactions/slash-commands.md

```diff
@@ -179,7 +179,39 @@ const data = new SlashCommandBuilder()
 			.setName('server')
 			.setDescription('Info about the server'));
 ```
+### Localizations
 
+Localized names and descriptions depending on the user's selected language.
+
+You can pick localizations from [this list](https://discord.com/developers/docs/reference#locales)
+```js {5-8,10-12,15,16}
+const { SlashCommandBuilder } = require('discord.js');
+
+const data = new SlashCommandBuilder()
+	.setName('dog')
+	.setNameLocalizations({
+		pl: 'pies',
+		de: 'hund',
+	})
+	.setDescription('Get cute picture of dog!')
+	.setDescriptionLocalizations({
+		pl: 'Słodkie zdjęcie pieska!',
+		de: 'Erhalten Sie nettes Bild des Hundes!',
+	})
+	.addStringOption(option =>
+		option
+            .setName('breed')
+            .setDescription('Breed of dog')
+		    .setNameLocalizations({
+			    pl: 'rasa',
+			    de: 'rasse',
+			})
+		    .setDescriptionLocalizations({
+		        pl: 'Rasa psa',
+		        de: 'Rasse des Hundes',
+		    })
```

Comment:
Is there a reason why the spacing is different between the first two and last two here?

Reply:
Seems to be mistake from [vvito7 suggestion](https://github.com/discordjs/guide/pull/1133#discussion_r883087924)